docs/source/tools/ledger-api-test-tool/index.rst

```diff
@@ -129,13 +127,25 @@ To test your ledger in a CI pipeline, run it as part of your pipeline:
 
    .. code-block:: console
 
-     $ java -jar ledger-api-test-tool localhost:6865 2>&1 /dev/null
+     $ java -jar ledger-api-test-tool.jar localhost:6865 --all-tests --exclude=TimeIT
```

Comment:
The original snippet didn't recommend to run any specific suite, meaning that only the semantics tests would be run. I'd prefer to leave it like that and possibly have a section about the available tests and what tests to leave out. This of course, doesn't have to be done within the context of this PR.

Reply:
the original snippet actually doesn't work anymore (not sure if it ever did), but i do think we want to say run the entire suite of tests in CI (less timeIT) rather than just semantic for assurance of fully happy ledger behaviour

Reply:
Then I'd advise to add a full explanation on which tests to run, how and why. `LotsOfPartiesIT`, `TransactionScaleIT` and `TimeIT` should each be run separately because of either stress testing or changing the global state of a ledger. Given the behavior of the latter, spinning up a ledger explicitly for that is also advisable.

Reply:
i excluded those 3 and added explanation of why to the readme